.scrutinizer.yml

```diff
@@ -4,4 +4,4 @@ build:
             dependencies:
                 after:
                     - yarn buildnode
-                    - yarn buildgrammar
+                    - yarn buildparser
```

Comment:
shouldn't this be part of buildnode? included in node-builder.js.

Reply:
No, because buildparser is generating Praser.php and not some front- or back-end webpacked bundle.